### DISCUSSION TITLE
Read Apache Iceberg table

### DISCUSSION THREAD

**Discussion by nrage2** [ClickHouse/ClickHouse#45611] [Q&A] [answered]
Is there any guidance on when reading from Apache Iceberg tables will be supported.   We have a data lake based on Iceberg and need to query this data set from within ClickHouse.

Thank you!

**Comment by UnamedRus** [comment, marked as answer]
Pr is https://github.com/ClickHouse/ClickHouse/pull/45384

Merging (after pr completion) can take up to several months. 
In mean time, you can test this build on your data lake.

![image](https://user-images.githubusercontent.com/9449405/214842482-c43aa04b-0121-490c-bf89-63a15c660b21.png)

**Reply by nrage2** [reply]
Thank you!

**Reply by UnamedRus** [reply]
It should be **ClickHouse build check** instead of special build check
